recipes/sortmerna/4.2.0/meta.yaml

```diff
@@ -0,0 +1,36 @@
+#
+# FILE: meta.yaml
```

Comment:
Is there a particular reason for having the `meta.yaml` file in a versioned subdirectory? Normally, bioconda only keeps a single file for the most recent version.

Reply:
None in my view. Just following my predecessors who I hope knew that reason.

Reply:
It appears this was done in PR #940 at a time when bioconda policies were less well established.  So I suggest to clean this up by following the currently recommended [best practice](https://bioconda.github.io/contributor/guidelines.html#versions), and to move the `meta.yaml` one dir up.

Reply:
I'm going to merge all existing directories into a single one and use git tags to distinguish between the releases